src/components/App/App.tsx

```diff
@@ -49,16 +48,11 @@ function App() {
     console.log('inside handleSubmit', selectedCountry);
     getCurrentCountry()
   }
- 
+
   useEffect(() => {
-    if (!countries.length) {
```

Comment:
Nice job eliminating the conditional from the useEffect! Much cleaner to just add the [] so that the function will run once on page load